userChrome.css

```diff
@@ -669,6 +669,10 @@
     /* Original: 0.5em */
     padding-block: var(--menu-padding) !important;
   }
+  menupopup > menu {
+    padding-left: 0.5em;
+    padding-right: 0.5em;
```

Comment:
Could you remove `padding-right`?
With this line, the right side of `Context Menu`(`Win10`) looks wide.

Menu's `arrow` is planning to completely right-align for consistency.
(Panel, Other menus ..etc)

Reply:
After merging, I will take care of it.